client/components/repository/index.vue

```diff
@@ -34,6 +34,31 @@ import selectActivity from '@/components/repository/common/selectActivity';
 import sortBy from 'lodash/sortBy';
 import withUserTracking from 'components/common/mixins/userTracking';
 
+const getTabItems = ({ hasWorkflow, isAdmin, isRepositoryAdmin, hasActivities }) => {
+  return [
+    {
+      name: 'Structure',
+      route: 'repository',
+      icon: 'file-tree'
+    },
+    hasActivities && hasWorkflow && {
+      name: 'Progress',
+      route: 'progress',
+      icon: 'chart-timeline-variant'
+    },
+    hasActivities && {
+      name: 'History',
+      route: 'revisions',
+      icon: 'history'
+    },
+    (isAdmin || isRepositoryAdmin) && {
```

Comment:
Since isAdmin and isRepositoryAdmin now have a single use, consider passing something like hasSettingsAccess to make the function signature leaner :bowtie:

Reply:
:+1: